draft-ietf-quic-manageability.md

```diff
@@ -1023,6 +1023,36 @@ not arrive at their destination. Admitting a few packets allows the QUIC
 endpoint to determine that the path accepts QUIC. Sudden drops afterwards will
 result in slow and costly timeouts before abandoning the connection.
 
+## Guiding Path MTU
+
+Some networks are required to be able to carry 1500-byte packets but become
+inefficient at this size, because such packets require fragmentation and
+reassembly across a segment where the effective MTU is smaller (e.g. due to
+IPSEC overhead). This leads to a need to influence a majority of senders to
+use smaller packets, so that the limited reassembly capacity is not exceeded.
```

Comment:
It's worth calling out that these networks fragment instead of dropping v4 packets with DF set and IPv6 packets(assuming that's what they do)

Reply:
Yes happy to make this clearer in the text.